### ISSUE TITLE
584 accumulation and frequency of use note

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#235]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/5XX.csv

**Comment by cspayne** [comment]
Response to my question in [Student/Project Management Discussions](https://github.com/uwlib-cams/MARC2RDA/discussions/433#discussioncomment-7857235) from @GordonDunsire 
> I agree that 584 seems to be applicable to collections only.
> 
> More accurately, 584 is applicable to a collection Work:
> 
>     $a Accumulation: maps to rdaw:P10368 "has frequency"
>     $b Frequency of use: is way out of scope for RDA and nearly every cataloguing code I know of
>     $3 Materials specified: maps to rdaw:P10088 "has title of work"
>     $5 Institution to which field applies: maps to rdaw:P10631 "has collector corporate body"
>     $a + $b + $3 + $5: maps to rdaw:P10330 "has note on work"
> 
> $3 maps to a general title because the examples are of generic titles that would normally be augmented with the name of the holding institution. This can be made more specific by using $5:
> $5 + $3 maps to rdaw:P10223 "has preferred title of work".
> 
> The note concatenates the subfields in the order in which they appear; punctuation is added and replaced as required.
> 
> Example:
> 
> collectionWork1 rdaw:P10088 "General subject files" ;
> rdaw:P10223 "Institution A General subject files";
> rdaw:P10368 "45 cu. ft. average annual accumulation 1970-1979.";
> rdaw:P10368 "5.4 cu. ft. average monthly accumulation, 1979-82.";
> rdaw:P10368 "Current average month accumulation is 2 cu. ft.";
> rdaw:P10631 "Institution A";
> rdaw:P10330 "General subject files: 45 cu. ft. average annual accumulation 1970-1979; 5.4 cu. ft. average monthly accumulation, 1979-82; Current average month accumulation is 2 cu. ft." .
> 
> Other information about the collection work is taken from the rest of the MARC 21 record which needs to separate out the collection work from the collection manifestation. In other words, I think the existence of 584 in a record indicates that a special map for collections should be used in the transform.

I have a follow up question for subfield $3. This subfield seems to indicate that the accumulation and frequency of use note field is describing a subcollection. If $3 maps to "has title of work", such as in your example, does that mean that the IRI for the collectionWork (collectionWork1 in your example) is the IRI for this subcollection as opposed to the entire collection? If so, is there a good way to indicate this in the transformation notes?

**Comment by GordonDunsire** [comment]
@cspayne: Correct, the collection work is assumed to be a sub-collection if subfield $3 is present. I am guessing that the rest of the record describes the super-collection, but a default assumption is that values for the super-collection apply to the sub-collection unless otherwise specified. Are there any full MARC 21 records with this note to help us see the context? There are two scenarios, depending on the presence of subfield $3. If a super-collection and sub-collection work/manifestation are generated, they are linked with rdaw:P10147 "has part work" and rdam:P30033 "has part manifestation".

**Comment by tmqdeborah** [comment]
@cspayne and @GordonDunsire: In my file of 5.6 million LC records from Jan 2022, only 5 records contain a 584 field. A printout of those 5 records is attached. 
[AND584.txt](https://github.com/uwlib-cams/MARC2RDA/files/13694005/AND584.txt)


I have added the 584 as an identifying marker in my list for collection works.

Re. mapping $a Accumulation to rdaw:P10368 "has frequency": Is the "**Rate** at which the described materials are accumulating expressed as a ratio of volume to time period" (i.e., how much has the size of the collection grown over a certain time) really an indication of RDA Frequency ("An **interval** at which a part, issue, or iteration of a diachronic work is issued")? 

 

Re. the $3 ("Part of the described material to which the accumulation and frequency of use note applies"): I suggest that the $3 is only present if the sub-collection is not described separately, but the cataloger felt it was important to note that its accumulation and frequency of use was different from other 'parts' (sub-collections) of the super collection.

**Comment by GordonDunsire** [comment]
@tmqdeborah: Subfield $a is treated as an unstructured description of the frequency. A frequency is number per unit time; the question is, a number of what? For a standard diachronic work, the number is of issues or iterations. For a collection work, it can't be issues or iterations, so it must be parts. I think it is a reasonable interpretation that the parts in this case are timespan-defined sub-collections, for example the 1971 intake with an authorized access point of 'Institution A General subject files. 1971'. These sub-collections may, or may not, be functional within the holding library, but the beauty of sub-collections is that they are whatever you define them to be, within the mono-hierarchy constraint. In this case, we have no information except the hint of shelving location defining a sub-collection, so it's an unstructured description.

That's my judgement in the absence of further information. Other statements in the record may indicate a value that is a structured description, such as 'irregular'.

Alternatively, or in parallel, we can/should add boilerplate to the note, as we have decided to do for Manifestation:

collectionWork1 rdaw:P10330 "Accumulation: General subject files: 45 cu. ft. average annual accumulation 1970-1979; 5.4 cu. ft. average monthly accumulation, 1979-82; Current average month accumulation is 2 cu. ft." .

I think a simple "Accumulation" is sufficient for boilerplate, because frequency of use (subfield $b) is absurd :-(

Anyway, that's my theoretical stance. Thanks for extracting the real data; I'll have a look and take a reality check :-)
